ee/clickhouse/migrations/0002_events_materialized.py

```diff
@@ -1,7 +1,3 @@
 from infi.clickhouse_orm import migrations
 
-from ee.clickhouse.sql.events import EVENTS_WITH_PROPS_TABLE_SQL
-
-operations = [
-    migrations.RunSQL(EVENTS_WITH_PROPS_TABLE_SQL),
-]
+operations = [migrations.RunSQL("SELECT 1")]
```

Comment:
This seems like the wrong way to handle this given that the table would continue to exist on cloud and any already done installations.

It also points out a flaw in how we're writing migrations: We import code from app instead of having the migration code be self-contained. Meaning that if the code changes, migration changes retroactively.

I think the answer here is:
1. `operations = []` - SELECT 1 is pointless
2. New migration to drop table if exists.

Reply:
haha you and I think similarly. I originally had #1.

1. `operations = []` throws type errors unfortunately. It needs something.
2. New migration to drop table if it exists is great. I'll add that.

Reply:
You could then do `operations = [] # type: ignore`